engine/client/cl_scrn.c

```diff
@@ -342,12 +342,18 @@ SCR_BeginLoadingPlaque
 */
 void SCR_BeginLoadingPlaque( qboolean is_background )
 {
+#if !XASH_DEDICATED
```

Comment:
XASH_DEDICATED useless in `client` because client sources are never added in dedicated build.